tests/modules/sightings/resources/test_identify_sighting.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+import uuid
+from unittest import mock
+import tests.extensions.tus.utils as tus_utils
+import tests.modules.asset_groups.resources.utils as asset_group_utils
+import tests.modules.sightings.resources.utils as sighting_utils
+import tests.utils as test_utils
+
+
+def test_sighting_identification(
+    flask_app,
+    flask_app_client,
+    researcher_1,
+    internal_user,
+    test_root,
+    db,
+):
+    # pylint: disable=invalid-name
+    from app.modules.sightings.models import Sighting, SightingStage
+
+    asset_group_uuids = []
+    sighting_uuids = []
+    transactions = []
+    try:
+        # Create two sightings so that there will be a valid annotation when doing ID for the second one.
+        # Otherwise the get_matching_set_data in sightings will return an empty list
+        transaction_id, test_filename = tus_utils.prep_tus_dir(test_root)
```

Comment:
I wonder what we want failure to look like when there truly isn't an available matching set for the query.